packages/alchemy/package.json

```diff
@@ -23,23 +23,11 @@
   "exports": {
     ".": {
       "types": "./dist/types/index.d.ts",
-      "import": "./dist/esm/index.js",
-      "default": "./dist/cjs/index.js"
-    },
-    "./viem": {
-      "types": "./dist/types/viem/index.d.ts",
-      "import": "./dist/esm/viem/index.js",
-      "default": "./dist/cjs/viem/index.js"
+      "import": "./dist/esm/src/index.js",
+      "default": "./dist/cjs/src/index.js"
```

Comment:
I could not tell you what's different about aa-alchemy and aa-core that aa-alchemy is spitting out files into `src` for the esm and cjs files